### DISCUSSION TITLE
The Windows install ends prematurely

### DISCUSSION THREAD

**Discussion by ObiW4nKenobi** [rapid7/metasploit-framework#14727] [Q&A] [answered]
I've tried installing this twice on Windows 10 version 2004. Both times the install ended prematurely. So, I give up for now until I get some further feedback!

[https://i.imgur.com/zQUbBH2.png](url)

**Comment by bcoles** [comment, marked as answer]
Perhaps this will help. #12600

If installation fails you can check the `msf.log` log file for debug info.

**Reply by ObiW4nKenobi** [reply]
I do not find a log with that exact name. Is it pacman? There are like 40 files ending in .log.

**Reply by bcoles** [reply]
Refer to #12600

**Reply by ObiW4nKenobi** [reply]
Thank You, I didn't actually realize for awhile that your message was referring me to another thread!
